tuf-spec.md

```diff
@@ -568,7 +575,7 @@ repo](https://github.com/theupdateframework/specification/issues).
         PUBLIC is in PEM format and a string.
 
    The KEYID of a key is the hexdigest of the SHA-256 hash of the
-   canonical JSON form of the key.
+   metadata format encoding of the key.
```

Comment:
I'd prefer ... 
```suggestion
   canonical form of the key.
```
... akin to what you have for SIGNATURE. But I'm fine with this too.